packages/aws-rfdk/lib/deadline/test/secrets-management.test.ts

```diff
@@ -586,111 +587,30 @@ describe('SecretsManagementIdentityRegistration', () => {
-      //     ...baseProps,
-      //     dependent: client2,
-      //     vpcSubnets: { subnetGroupName: 'a' },
-      //   });
-
-      //   // THEN
-      //   const resolvedCalls = deploymentInstance.userData.addCommands.mock.calls.map(call => {
-      //     return deploymentInstanceStack.resolve(call);
-      //   });
-      //   const expectedSubnetIdOrder = ([] as string[]).concat(
-      //     ...vpc.selectSubnets({ subnetGroupName: 'a' }).subnetIds,
-      //     ...vpc.selectSubnets({ subnetGroupName: 'z' }).subnetIds
-      //   )
-      //   const expectedCall = [{
-      //     "Fn::Join": [
-      //       "",
-      //       [
-      //         // Command is run as "ec2-user" which has the database credentials stored
-      //         `sudo --login -u ec2-user ${identityRegistrationSettingsScriptLocalPath} `,
-      //         {
-      //           "Fn::Join": [
-      //             " ",
-      //             [
-      //               "--region",
-      //               {
-      //                 "Ref": "AWS::Region"
-      //               },
-      //               // The Deadline Secrets Management admin credentials secret ARN is passed
-      //               "--credentials",
-      //               deploymentInstanceStack.resolve(`"${repository.secretsManagementSettings.credentials!.secretArn}"`),
-      //               // The Render Queue's ALB subnets are passed as --connection-subnet args
-      //               ...vpc.selectSubnets({ subnetGroupName: RENDER_QUEUE_ALB_SUBNET_NAME }).subnetIds.map(subnetID => {
-      //                 return {
-      //                   'Fn::Join': [
-      //                     '',
-      //                     [
-      //                       '--connection-subnet "',
-      //                       deploymentInstanceStack.resolve(subnetID),
-      //                       '"'
-      //                     ],
-      //                   ],
-      //                 };
-      //               }),
-      //               // The Deadline Client's subnets, desired role, and registration status are passed as --source-subnet args
-      //               ...expectedSubnetIdOrder.map(subnetID => {
-      //                 return {
-      //                   'Fn::Join': [
-      //                     '',
-      //                     [
-      //                       '--source-subnet "',
-      //                       deploymentInstanceStack.resolve(subnetID),
-      //                       `,${baseProps.role},${baseProps.registrationStatus}"`,
-      //                     ],
-      //                   ],
-      //                 };
-      //               }),
-      //             ],
-      //           ],
-      //         },
-      //       ],
-      //     ],
-      //   }];
-      //   expect(resolvedCalls).toContainEqual(expectedCall);
-      // });
+      test('warns about dedicated subnets when ALB and source subnets match', () => {
```

Comment:
```suggestion
      test('warns about dedicated subnets when render queue ALB and source subnets match', () => {
```

this is a really minor nit though, so I won't hold the PR back for it

Reply:
Fixed